.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,26 @@
+name: github pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
+        with:
+          mdbook-version: '0.4.4'
+          # mdbook-version: 'latest'
```

Comment:
Is there a reason why `mdbook-version` is not set to `latest`?

Reply:
Because builds might break if there are breaking changes?